docs/source/tutorials/faq.ipynb

```diff
@@ -68,16 +68,26 @@
    "id": "3868ee8b",
    "metadata": {},
    "source": [
-    "### Why isn’t CleanLearning working for my data?"
+    "### Why isn’t CleanLearning working for me?"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "d13c9cd0",
    "metadata": {},
    "source": [
-    "At this time, CleanLearning only works directly with numpy matrices or pd.DataFrames.\n",
-    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities from your model via cross-validation and pass them as inputs."
+    "At this time, CleanLearning only works with data formatted as numpy matrices or pd.DataFrames, \n",
+    "and with models that are compatible with the `sklearn` API.\n",
```

Comment:
Add "Note skorch can be used for PyTorch compatibility. Similarly [KerasClassifier](https://www.tensorflow.org/api_docs/python/tf/keras/wrappers/scikit_learn/KerasClassifier) and [KerasRegressor](https://www.tensorflow.org/api_docs/python/tf/keras/wrappers/scikit_learn) can be used for Tensorflow/Keras compatibility."

Reply:
added replacing the latter two with scikeras. I think those two have been removed from public TF API in versions >=2.7:  https://www.tensorflow.org/versions/r2.7/api_docs/python/tf/keras/wrappers/scikit_learn

Reply:
Just FYI because of this removal and the fact that scikeras is missing some desirable features, I've actually added our keras wrapper to `cleanlab.experimental` in another PR: 

https://github.com/cleanlab/cleanlab/blob/3f29a5d3a73d04844fff6d3c11b1ab6244ae2670/cleanlab/experimental/keras.py